experimental/dds/tree2/api-report/tree2.api.md

```diff
@@ -2135,6 +2132,11 @@ export interface TreeContext extends ISubscribable<ForestEvents> {
     readonly schema: TreeSchema;
 }
 
+// @alpha
+export interface TreeCore<out TSchema = unknown> {
+    readonly schema: TSchema;
```

Comment:
@noencke - I was thinking that your 'is()' function should be updated, but I'm having trouble finding the declaration.  Is it still around?

@CraigMacomber - Woah...  covariance!

Reply:
It is, it's in the proxies.ts file. I was actually working on another PR right now to add it to the node() API - But this will do instead!

Reply:
(as in - I will add it to TreeNodeCore)